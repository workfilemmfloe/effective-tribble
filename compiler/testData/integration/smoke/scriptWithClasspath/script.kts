import javax.inject.Inject

println(Inject::class.java)
