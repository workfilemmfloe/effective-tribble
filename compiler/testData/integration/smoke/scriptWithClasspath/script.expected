OUT:
interface javax.inject.Inject

Return code: 0
